conf/test.config

```diff
@@ -41,4 +51,25 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
+
+    withName: GOAT_TAXONSEARCH {
+         ext.args = '-l -b'
+     }
+
+    withName: BUSCO {
+         ext.args = '--mode genome'
+     }
+
+    withName: CHUNK_FASTA_BUSCO {
+          ext.args = '--chunk=100000 --overlap=0 --max-chunks=10 --min-length=1000'
+     }
+
+    withName: DIAMOND_BLASTX {
+           ext.args = '--evalue 1.0e-10 --max-target-seqs 10 --max-hsps 1'
+     }
+
+    withName: UNCHUNK_BLASTX {
+            ext.args = '--count 10'
+     }
```

Comment:
Same as in #28 . This should be in `conf/modules.config`

Reply:
Done.